docs/tutorials/writing-sd-card-image-etcher.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Writing an SD Card Image Using Etcher (Cross-Platform)
+subject: SD Card Image
+---
+
+1. Download and install Etcher from [their website](http://etcher.io).
+2. Launch Etcher after it has finished installing.
+
+   <div>{% include screenshot.html source="/images/etcher/home.png" caption="Etcher home screen" %}</div>
```

Comment:
I see. Maybe this would be better handled with a javascript fixup à la `javascripts/style-helpers.js` so that we don't have to remember to fix it up with `<div>` tags every time we use the screenshot include.